twake/backend/node/src/services/documents/services/index.ts

```diff
@@ -7,12 +7,15 @@ import { CrudException } from "../../../core/platform/framework/api/crud-service
 import { PublicFile } from "../../../services/files/entities/file";
 import { getDefaultDriveItem, getDefaultDriveItemVersion } from "../utils";
 import { FileVersion, TYPE as FileVersionType } from "../entities/file-version";
+import gr from "../../global-resolver";
+import { hasCompanyAdminLevel, isCompanyAdminRole } from "src/utils/company";
```

Comment:
```suggestion
import { hasCompanyAdminLevel, isCompanyAdminRole } from "src/utils/company";
```
i think this is causing CI errors, because the src folder can't be resolved in tests, should be `../../../utils/company`